src/Symfony/Bundle/FrameworkBundle/Resources/config/asset_mapper.php

```diff
@@ -167,7 +168,7 @@
                 service('asset_mapper'),
                 service('asset_mapper.importmap.config_reader'),
                 service('asset_mapper.importmap.remote_package_downloader'),
-                service('asset_mapper.importmap.resolver'),
+                service('asset_mapper.importmap.resolver_registry'),
```

Comment:
I'd suggest using a strategy/facade resolver with the same interface.. i don't think this registry should be exposed.. Maybe it would be better to mimic Notifier Transports class ?

Reply:
I will check the Notifier Transports implementation then.

I exposed it here as I didn't have found another way to allow the `ImportMapManager` to pick the right resolver depending on the package 😅